composer.json

```diff
@@ -32,7 +32,8 @@
     "require-dev": {
         "cakephp/cakephp-codesniffer": "^4.0",
         "cakephp/authorization": "^2.0",
-        "phpunit/phpunit": "^8.0"
+        "phpunit/phpunit": "^8.0",
+        "psalm/phar": "^3.6"
```

Comment:
This should match the version used in GA. We should remove psalm-setup if we're adding it here.

Reply:
Whoops this isn't supposed to be here.